pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -3041,7 +3043,7 @@ var _ = Describe("Converter", func() {
 			Expect(domainSpec.OS.BootLoader.Secure).To(Equal(secureLoader))
 			Expect(path.Base(domainSpec.OS.BootLoader.Path)).To(Equal(efiCode))
 			Expect(path.Base(domainSpec.OS.NVRam.Template)).To(Equal(efiVars))
-			Expect(domainSpec.OS.NVRam.NVRam).To(Equal("/tmp/mynamespace_testvmi"))
+			Expect(domainSpec.OS.NVRam.Source.File).To(Equal("/var/lib/libvirt/swtpm/testvmi_VARS.fd"))
```

Comment:
Don't we have a similar test for non-root VMs also?

Reply:
Added